config/common/traces-limits-v2.toml

```diff
@@ -84,7 +84,7 @@ BLAKE               = 0
 POINT_EVAL          = 0
 
 # Introduced from beta-v4.0-rc6-CANCUN and present on Cancun
-BLS_DATA                = 2147483647
-RLP_UTILS               = 2147483647 # Note: need to confirm if this limit could have the same limit as RLP_TXN
+BLS_DATA                = 8192
```

Comment:
@Filter94 why change these two from max int to 8192, same as `TXN_DATA`? `BLS_DATA` would be always 15 from Cancun

Reply:
I'll set it to `1073741824`. It needs to be a power of 2